sdk/storage/azure-storage-file-share/tests/test_file.py

```diff
@@ -1446,6 +1446,8 @@ def test_update_range_from_file_url_when_source_file_does_not_have_enough_bytes(
             source_file_client.share_name,
             source_file_client.file_path,
             source_file_client.credential.account_key,
+            FileSasPermissions(read=True),
```

Comment:
This comment applies to both here and async

This test was giving me test failures, but the test was probably not working as intended. 
It was passing by hitting a `HttpResponseError`, but the error was `CannotVerifyCopySource` which means the copy source couldn't be accessed. Upon further investigation, this test as it stands doesn't provide permissions or expiry, which means the access issue is due to a not having a fully formed SAS token, and our input validation actually caught this case 😄 So I fixed it up and just re-recorded them and now they fail for the right assert!